source/rules/roles.txt

```diff
@@ -43,25 +48,21 @@ What are Roles?
 ~~~~~~~~~~~~~~~
 
 A **role** is a named set of permissions that a user may have for documents in a
-MongoDB collection. Each role represents a possible relationship between a user
-and an individual document. A user may play at most one role per document in
-a request.
+MongoDB collection. A role has an "apply when" :ref:`expression <expressions>`
```

Comment:
Why not GUI label: it's not necessarily something in the UI - it could be something set from a configuration file.

Why not code: I prefer to minimize the use of monospace and in this case it's describing the concept rather than the specific instance of it in the configuration file.

So, yeah, my preference is for quotation marks. I can go through and check for consistency.